Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -624,6 +637,17 @@ open class ChatMessageListVC:
         return !message.isBlank
     }
 
+    private func isImagePreview(_ message: ChatMessage?) -> Bool {
```

Comment:
@parth-simformsolutions  it is more convenient if you add this method to ChatMessage class inside extension , thanks